### ISSUE TITLE
Using foreign tables

### ISSUE THREAD

**Issue by mustafaakin** [electric-sql/pglite#10]
I know it might be early to ask but it'd be great to have support for it and use pglite as a wrapper for running SQL over other resources securely

**Comment by kiwicopple** [comment]
if this goes ahead we're be open to adding Wrappers as a single interface for many other data stores:

https://github.com/supabase/wrappers

That said, I think this feature request should be delayed as long as possible while the current storage mechanisms reach maturity.

**Comment by mustafaakin** [comment]
Yeah it'd be great. I was thinking of something like Steampipe but a permissive & compatible license.

**Comment by samwillis** [comment]
@kiwicopple Supabase wrapper support would be awesome, but as you said priority is stability, parameterize queries, storage, then extensions. Support for opening the same db from multiple tabs is also essential (probably by routing all operations to a single worker with a leader electrician, should make an issue to this...)

**Comment by metduman80** [comment]
Wow, this would be a great feature to have. Imagine creating something similar to https://steampipe.io/.
